examples/performance/README.md

```diff
@@ -26,6 +26,10 @@
     * `cire-repeats-inv` (int, 1): control detection of dimension-invariants
     * `cire-mincost-inv` (int, 50): minimum cost of a dimension-invariant candidate
     * `cire-onstack` (boolean, False): control scope of temporary arrays (on the heap by default)
+  * Device-specific:
+    * `gpu-fit` (boolean, False): control movement of saved TimeFunctions between host and device
```

Comment:
Not clear from comment why this is 'fit'

Reply:
OK, improving